go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/ElrondNetwork/elastic-indexer-go
+
+go 1.15
+
+require (
+	github.com/ElrondNetwork/elrond-go v1.1.27
+	github.com/ElrondNetwork/elrond-go-logger v1.0.4
+	github.com/elastic/go-elasticsearch/v7 v7.10.0
+	github.com/stretchr/testify v1.7.0
+)
+
+replace github.com/ElrondNetwork/elrond-go => /home/mihai/go/src/github.com/ElrondNetwork/elrond-go
```

Comment:
do we need this replace?

Reply:
for the moment I cannot reference a commit hash from branch `remove-indexer ` so I have to use a local version will be removed.

Reply:
right

Reply:
removed this replace.